book/interpretation.qmd

```diff
@@ -778,83 +515,48 @@ Looking at the plots above, we can read that the biggest contributors to the fin
+Shapley values can take a long time to compute.
+This process can be sped up at the expense of accuracy.
+The parameters `B` and `N` can be used to tune this trade-off, where `N` is the number of observations on which conditional expectation values are estimated (500 by default) and `B` is the number of random paths used to calculate Shapley values (25 by default).
 :::
 
-#### Ceteris Paribus Effects
-
-In the previous section, we have introduced a global explanation -- partial dependence plots. Ceteris paribus plots are the local version of that explanation. They show the response of a model when only one feature is changed while others stay unchanged. Ceteris paribus profiles are also known as individual conditional explanations (ICE). More on this technique can be found in @sec-feature-effects and the chapter *Ceteris-paribus Profiles* of the [EMA book](https://ema.drwhy.ai/).
-
-The function `r ref("DALEX::predict_profile()")` calculates ceteris paribus profiles which can be visualized with the generic `plot()` function. The blue dot stands for the prediction for Steve.
+Finally, we can plot ICE curves using `r ref("DALEX::predict_profile()")`:
 
 ```{r interpretation-029, warning=FALSE}
 #| fig-height: 5
 #| fig-width: 8
 #| out-width: 90%
-#| fig-cap: Graphical summary of local model response profiles calculated by the ceteris-paribus method for three selected variables (age, amount, duration). Each variable is shown in a separate panel. 
-#| fig-alt: Graphical summary of local model response profiles calculated by the ceteris-paribus method for three selected variables (age, amount, duration). Each variable is shown in a separate panel. 
+#| label: fig-dalex-ice
+#| fig-cap: Individual conditional explanations (aka Ceteris Paribus) plots for 10 rows in the credit data (including Steve) for three selected variables (age, amount, duration).
+#| fig-alt: Plots are very similar to the PD plots from earlier except with 10 lines plotted. y-axis is 'prediction' ranging from 0.1 to 0.5. Left plot is titled 'age' and shows a dip in prediction value then increase between 20,30,70. Middle plot is titled 'amount' and shows a fairly linear increase in prediction value from 0.2 to 0.5. Right plot is titled 'duration' and show an increase in prediction value from 0.2-0.35 and then is flat. There is fairly good agreement between plotted curves.
 
-ranger_ceteris = predict_profile(ranger_exp, steve)
-plot(ranger_ceteris)
+plot(predict_profile(ranger_exp,  credit_x[240:250, ]))
 ```
 
-::: {.callout-tip}
-Ceteris paribus profiles are often plotted simultaneously for a set of observations. This helps to see potential deviations from the additivity of the model. To draw profiles for multiple observations, simply point to more rows in the data for the `predict_profile` function. Deviation from additivity is manifested by profiles that are not parallel.
-:::
-
-## Training or Test Data? {#sec-dataset}
```

Comment:
I would leave at least a short discussion on this in this chapter.

Reply:
It is [here](https://github.com/mlr-org/mlr3book/pull/609/files#diff-3f7f824dec751eaf091aa8627c4f9f43bd07ddd9e4a78bab420f0e806e1cc5faR111).